app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_split = command.split()
+
+    with (
```

Comment:
Make sure that the command includes `cp` in the right place and that there are exactly two filenames that are passed in a command as mentioned [in the checklist](https://github.com/MykolaKrasulin/py-copy-file/blob/develop/checklist.md#code-efficiency).